parsec/backend/memory/factory.py

```diff
@@ -22,7 +25,9 @@
 
 @asynccontextmanager
 async def components_factory(config: BackendConfig, event_bus: EventBus):
-    (send_events_channel, receive_events_channel) = trio.open_memory_channel(math.inf)
+    (send_events_channel, receive_events_channel) = trio.open_memory_channel[
+        Tuple[str, Dict[str, object]]
```

Comment:
I have the feeling something's broken here: we use consider event have `str` here (and in `_send_event/_dispatch_event`) this is probably because it must be serialized to be send over postgresql listen/notify system. However the event bus is supposed to receive an Enum.

My guess is we should be bitten by this once this PR and #1680 are merged

Reply:
9ddb38c5be108a3fc89d6ff4ab2404d92401c0ed